docs/src/utils/componentInfo.ts

```diff
@@ -0,0 +1,73 @@
+import { ComponentType } from 'react'
+
+export interface IComponentInfoObjectTag {
+  name: string
+  title: 'accessibility' | 'example' | 'param' | 'returns' | 'see'
+  description: string
+  type: {
+    type: string
+    name: string
+  }
+}
+
+export interface IComponentInfoObjectProp {
+  type: 'bool' | 'component' | 'custom' | 'func' | 'node' | 'string' | 'enum'
+  name: string
+  description: string[]
+  defaultValue: string
+  value?: any[] // that is, allowed enum values
+  tags: IComponentInfoObjectTag[]
+  required: boolean
+}
+
+export interface IComponentInfoObject {
+  displayName: string
+  props: IComponentInfoObjectProp[]
+  constructorName: string
+  type: string
+  isParent: boolean
+  isChild: boolean
+  parentDisplayName: string | null
+  subcomponentName: string | null
+  subcomponents: string[]
+  apiPath: string
+  componentClassName: string
+  docblock: {
+    tags: IComponentInfoObjectTag[]
+    description: string[]
+  }
+  repoPath: string
+  filename: string
+  filenameWithoutExt: string
+}
+
+interface IComponentInfo extends RequireContextReturn {
```

Comment:
this one should be exported as well, due to instance of this interface is exported. Otherwise this might cause problems when this library will be integrated into other project